t/randomlib/TestsFor/A.pm

```diff
@@ -0,0 +1,9 @@
+package A;
+
+use Test::Class::Moose;
+
+sub test_a {
+    ok( 1, "package A");
```

Comment:
I know this won't really be run but I prefer single quotes for non-interpolated strings.